airflow/bin/cli.py

```diff
@@ -837,6 +837,12 @@ class CLIFactory:
             'help': "Runs a shell to access the database",
             'args': tuple(),
         },
+        {
+            'func': lazy_load_command('airflow.cli.commands.db_command.check'),
+            'name': 'check',
+            'help': "Chack if the database can be reached.",
+            'args': tuple(),
+        },
```

Comment:
We have the `/health` endpoint already, do you think that would cover this and we won't need a new command?

Endpoint: https://github.com/apache/airflow/blob/master/airflow/www/views.py#L182-L210

>An endpoint helping check the health status of the Airflow instance, including metadatabase and scheduler.

Reply:
But we do not run webserver nor airflow at this point yet. It's just environment we check.